docs/roles/secretary.rst

```diff
@@ -9,22 +9,38 @@ This page explains more about the responsibilities and role of the secretary.
 Description
 -----------
 
-< general description of role, 1-2 paragraphs >
-
+The secretary is responsible for record maintenance and for keeping RITlug organized.
+Their primary tool as an eboard member is the Runbook.
+Part of their role is to monitor the Runbook and ensure current practices by RITlug eboard members are accurately documented in the Runbook.
+They do not exclusively write documentation, but work with other eboard members to identify areas of need.
+Additionally, the secretary also builds the agenda for eboard meetings.
+This role can be considered the "librarian" or "information architect" of RITlug.
 
 Eligibility requirements
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-< basic eligiblity requirements – what if someone is on co-op? away from campus? working remote? is under academic suspension (i.e. low GPA)? >
+* Must be a full time student on main campus for at least one semester
+* Cannot be on academic probation
+* Must be present for **75% of weekly club meetings and eboard meetings**
 
 
 Responsibilities
 ----------------
 
-< responsibilities and routine work that someone in this position is directly accountable for – what do they do and what should other eboard members expect from this person? >
+* *Weekly Presentations*
+   * Review of presentations
+   * Addition of presentations to RITlug website
+* Set up sign in form for upcoming meetings
+* Set up live stream or video conference for meetings and presentations that require remote participants or viewers
+* Record meeting minutes during eboard meetings
+* Prepare the agenda for eboard or special meetings
+* Manage calendar events for meetings
+* Manage mailing list participants
```

Comment:
I would either heavily reword this, or just remove it entirely. 

Once participants on on the mailing list, there is no reason to do anything further.

Reply:
After thinking about it, I also agree with @Tjzabel to omit this line since it is unclear.